app/views/restaurants/new.html.haml

```diff
@@ -0,0 +1,31 @@
+%h2 Restaurant Registration Form
+-#
+= form_for :restaurant, :url => {:controller => 'restaurants', :action => 'create'} do |f|
+  .field
+    = f.label :name, "Restaurant Name"
+    %br/
+    = f.text_field :name, autofocus: true
+  .field
+    = f.label :address, "Address"
+    %br/
+    = f.text_field :address
+  .field
+    = f.label :city, "City"
+    %br/
+    = f.text_field :city
+  .field
+    = f.label :state, "State"
+    %br/
+    = f.text_field :state
+  .field
+    = f.label :country, "Country"
+    %br/
+    = f.text_field :country
+  .field
+    = f.label :rest_category_id, "Restaurant Category"
+    %br/
+    = f.collection_select(:rest_category_id, RestCategory.all, :id, :name)
```

Comment:
I usually make database calls in the controller and pass them in to the view. In this case I would add:
```ruby
@categories = RestCategory.all
```
to the `create` method in your `restaurants_controller` and use that collection here:
```haml
= f.collection_select(:rest_category_id, @categories, :id, :name)
```